src/style.css

```diff
@@ -1,3 +1,87 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  --color-link-hover: #00acdc;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  width: 1200px;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-self: center;
+}
+
+.menu {
+  height: 60px;
+}
+
+.nav-items {
+  display: flex;
+  align-items: center;
+  list-style: none;
+}
+
+.nav-items .nav__item {
+  margin-left: 20px; /* Adjust as needed */
+}
+
+.nav-items .nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav-items .nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav-items .nav__item .nav__link {
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  line-height: 60px;
+  position: relative;
+  display: block;
+}
+
+.nav-items .nav__item .nav__link:hover {
+  color: var(--color-link-hover);
+}
+
+.nav-items .nav__item .nav__link.is-active {
+  color: var(--color-link-hover);
+  height: 60px;
+}
+
+.nav-items .nav__item .nav__link.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: var(--color-link-hover);
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+@media screen and (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
 }
```

Comment:
Redundant
```suggestion
```

Reply:
what do u mean?

Reply:
don't understand what do u mean to do with @media